docs/reference/tctl/workflow/show.md

```diff
@@ -0,0 +1,57 @@
+---
+id: tctl-workflow-show
```

Comment:
```suggestion
id: show
```

The current id would produce the following slug: `/docs/reference/tctl/workflow/tctl-workflow-show`

Will look super clean as just `/docs/reference/tctl/workflow/show` no?

Reply:
Definitely. I was having a running disagreement with prettier (which produced yesterday's deluge of commits), but I think we've come to an understanding. The next commit will have the expected slugs.